wrapanapi/systems/ec2.py

```diff
@@ -1471,3 +1471,39 @@ def set_sns_topic_target_for_all_cw_rules(self, topic_arn):
             return True
         except Exception:
             return False
+
+    def import_snapshot(self, s3bucket, s3key, format="vhd", description=None):
+        self.logger.info(
+            " Importing snapshot %s from %s bucket with description %s in %s started successfully.",
+            s3key, s3bucket, description, format
+        )
+        try:
+            result = self.ec2_connection.import_snapshot(
+                DiskContainer={
+                    'Description': description if description is not None else s3key,
```

Comment:
*Optional*

Suggest using pythonic `description or s3key`, as it'll automatically if-else when description is None